pkg/kubelet/volumes.go

```diff
@@ -193,14 +192,22 @@ func (kl *Kubelet) getPodVolumes(podUID types.UID) ([]*volumeTuple, error) {
 	return volumes, nil
 }
 
+// cleanerTuple is a union struct to allow separating detaching from the cleaner.
+// some volumes require detachment but not all.  Cleaner cannot be nil but Detacher is optional.
+type cleanerTuple struct {
+	Cleaner  volume.Cleaner
```

Comment:
`cleanerTuple` here can be renamed to `cleaner` after the big rename in #19599.

There's `volumeTuple` in this file, so there is some precedent for this naming convention.